test/extensions/filters/listener/proxy_protocol/proxy_protocol_test.cc

```diff
@@ -994,6 +999,8 @@ TEST_P(WildcardProxyProtocolTest, BasicV6) {
   disconnect();
 }
 
+// TODO (Jojy): Add tests for UDP ?
```

Comment:
I'd take this out (and not bother renaming the tests yet).

Unfortunately as far as I know there's no great way to do this for UDP proxying. In QUIC, we can't insert the logical equivalent of the proxy header because transparent proxies aren't guaranteed to have the certs, so can't terminate the transport to insert things in.  There are non-RFCified super-hacky packet-munging tricks you can play with for QUIC when we get there, but it's far enough down the line I don't think we need a TODO for it.